build.sh

```diff
@@ -18,14 +18,33 @@ if [ $# -lt 1 ]; then
   echo "- linux-armv6"
   echo "- linux-armv7"
   echo "- linux-arm64v8"
+  echo "- darwin-x64"
   echo
   exit 1
 fi
 VERSION_VIPS="$1"
 PLATFORM="${2:-all}"
 
+# macOS
+# Note: we intentionally don't build these binaries inside a Docker container
+if [ $PLATFORM = "darwin-x64" ]; then
+  export VERSION_VIPS
```

Comment:
It might be worth adding a check that `uname -s` is "Darwin" here in case someone tries to run this on Linux.

Reply:
Good idea! Added with commit https://github.com/lovell/sharp-libvips/commit/3dbab0f89b50f3126bc43928663e782847fdfae8.